src/generator/operations.ts

```diff
@@ -652,11 +683,12 @@ function generateResponseUnmarshaller(op: Operation, response: Response, imports
     if ((mediaType === 'XML' && schemaResponse.schema.type === SchemaType.Array) || schemaResponse.schema.language.go!.discriminatorInterface) {
       target = 'result';
     }
-    unmarshallerText += `\treturn &result, resp.UnmarshalAs${getMediaFormat(response.schema, mediaType, `&${target}`)}\n`;
+    unmarshallerText += `\terr := resp.UnmarshalAs${getMediaFormat(response.schema, mediaType, `&${target}`)}\n`;
```

Comment:
Why does this need to be separated now?

Reply:
It broke the case for multiple responses, where the return type is `interface{}`.  Might be a bug in Go, I didn't investigate further.

Reply:
Ok good to know 👍